ibis/tests/sql/test_compiler.py

```diff
@@ -220,18 +220,18 @@ def test_scalar_exprs_no_table_refs(self):
             result = Compiler.to_sql(expr)
             assert result == expected
 
-    def test_expr_list_no_table_refs(self):
-        exlist = ibis.api.expr_list(
-            [
-                ibis.literal(1).name('a'),
-                ibis.now().name('b'),
-                ibis.literal(2).log().name('c'),
-            ]
-        )
-        result = Compiler.to_sql(exlist)
-        expected = """\
-SELECT 1 AS `a`, now() AS `b`, ln(2) AS `c`"""
-        assert result == expected
+    #     def test_expr_list_no_table_refs(self):
```

Comment:
I think it's fine to remove this code instead of commenting it out since we're explicitly removing the API in this PR.